packages/video_player/video_player/lib/video_player.dart

```diff
@@ -230,7 +230,7 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// null.
   /// **Android only**: The [formatHint] option allows the caller to override
   /// the video format detection code.
-  /// [httpHeaders] option allows to specify HTTP headers
+  /// [httpHeaders] option allows to specify HTTP headers and optionally, the User-Agent
```

Comment:
This change is confusing; it makes it sound like the user agent is separate from the headers, but there is no such option. It's not at all clear to a reader how this comment maps to the parameters.